src/grid/initial_conditions.cpp

```diff
@@ -582,12 +583,12 @@ void Grid3D::Riemann(parameters const &P)
 #endif  // SCALAR
 #ifdef DE
             C.GasEnergy[id] = P.P_r / (gama - 1.0);
-#endif  // DE
-          }
-        }
-      }
-    }
-  }
+#endif       // DE
+          }  // if diaph
```

Comment:
I don't think we should have labels like this. They're likely not to stay up to date and most IDEs/editors include color coding to help with which brackets belong with which.

Reply:
Removed in [df7df09](https://github.com/cholla-hydro/cholla/pull/315/commits/df7df09ab36ce0757abec64009eb05af275dbeef)